libs/coin-framework/src/account/helpers.ts

```diff
@@ -69,6 +69,13 @@ export const isAccountEmpty = (a: AccountLike): boolean => {
       (a as any).balance.isZero()
     );
   }
+  if (a.type === "Account" && a.currency.family === "solana") {
+    return (
+      (a as any).solanaResources &&
+      (a as any).balance.isZero() &&
```

Comment:
wouldn't we want delegation to be disabled if balance isZero AND spendableBalance isZero ?